server/opendp_apps/analysis/tools/dp_histogram_spec.py

```diff
@@ -0,0 +1,175 @@
+from opendp.trans import *
+from opendp.meas import *
+from opendp.core import *
+from opendp.typing import *
+
+enable_features("floating-point")
+
+from opendp_apps.analysis.tools.stat_spec import StatSpec
+
+
+class DPHistogramSpec(StatSpec):
+    """
+    Initiate with dict of properties. Example of needed properties:
+
+    spec_props = dict(var_name="hours_sleep",
+                      col_index=3,
+                      variable_info=dict(min=0, max=24, type=VAR_TYPE_FLOAT),
+                      statistic=DP_HISTOGRAM,
+                      dataset_size=365,
+                      epsilon=0.5,
+                      ci=CI_95.
+                      fixed_value=1)
+    """
+    def __init__(self, props: dict):
+        """Set the internals using the props dict"""
+        # TODO: Generates an error when this isn't initialized somewhere
+        self.accuracy_msg = ""
+        super().__init__(props)
+
+    def additional_required_props(self):
+        """
+        Add a list of required properties
+        example: ['min', 'max']
+        """
+        return ['min', 'max']
```

Comment:
@ecowan I added this line, let me know if it's correct. 
- It will make sure `min`/`max` are valid. 
- if `missing_values_handling` is "insert fixed", it checks that the `fixed_value` is numeric and not outside min/max

Reply:
Sounds good